openshift-ci/Dockerfile.metadata

```diff
@@ -0,0 +1,17 @@
+# see: https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md
+FROM scratch
+
+ARG CHANNEL=4.6
+ARG FULL_OPERATOR_IMAGE=quay.io/openshift-kni/performance-addon-operator-metadata:4.6-snapshot
+ARG OLM_SOURCE=deploy/olm-catalog
+ARG OLM_METADATA=deploy/metadata
+
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=performance-addon-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=4.6.0
+LABEL operators.operatorframework.io.bundle.channel.default.v1=4.6.0
+
+COPY ${OLM_SOURCE}/performance-addon-operator/${CHANNEL}.0 /manifests/
```

Comment:
we need to replace FULL_OPERATOR_IMAGE here in the csv, otherwise the pullspec will be broken. But since this image is FROM SCRATCH we can't do it in the Dockerfile.